source/rst/lss_examples.rst

```diff
@@ -1,36 +1,36 @@
 
-Methods of the Linear State Space Class
+Methods of the LinearStateSpace Class
 =======================================
 
 QuantEcon contains a class which has various methods for operating on
-Linear State Space Models (such models are explained in detail `in this
-lecture <https://lectures.quantecon.org/py/linear_models.html>`__).
+``LinearStateSpace`` Models (such models are explained in detail `in this
+lecture <https://lectures.quantecon.org/py/linear_models.html>`__)
 
 In this notebook, we will illustrate the methods that can be used on
-instances of this class.
+instances of this class
 
 To illustrate these methods, we will use Paul Samuleson's (1939)
```

Comment:
We can also add a citation here with `:cite:`` ` - you'll need to edit the `quant-econ.bib` file